Tests/TestHelpers.swift

```diff
@@ -113,14 +114,23 @@ enum HTTPBin: TargetType {
         switch self {
         case .basicAuth:
             return "{\"authenticated\": true, \"user\": \"user\"}".data(using: String.Encoding.utf8)!
-        case .post, .upload, .uploadMultipart:
+        case .post, .upload, .uploadMultipart, .validatedUploadMultipart:
             return "{\"args\": {}, \"data\": \"\", \"files\": {}, \"form\": {}, \"headers\": { \"Connection\": \"close\", \"Content-Length\": \"0\", \"Host\": \"httpbin.org\" },  \"json\": null, \"origin\": \"198.168.1.1\", \"url\": \"https://httpbin.org/post\"}".data(using: String.Encoding.utf8)!
         }
     }
 
     var headers: [String: String]? {
         return nil
     }
+
+    var validationType: ValidationType {
+        switch self {
+        case .validatedUploadMultipart:
```

Comment:
hmm, how about adding `customCodes` as an associated value, so we can provide it in tests? we could also comment in tests _why_ do we use this constant

Reply:
I can do that 👌. To be honest the reason why this is such a hack of a test is I can't validate a stubbed request. Or else I wouldn't need another enum case 😕